src/api/passport.js

```diff
@@ -0,0 +1,23 @@
+import bcrypt from 'bcryptjs';
+
+export const validatePassword = function (passport, password, next) {
+  bcrypt.compare(password, passport.password, next);
+};
+
+export async function hashPassword(password) {
+  var salt = 8;
+
+  if (password) {
+    return new Promise((resolve, reject) => {
+      bcrypt.hash(password, salt, function (err, hash) {
+        if (err) {
+          console.error(err);
```

Comment:
Do we have a "koa" or app logger somewhere that we can reuse ?

Reply:
yes fixed that! thanks